common-libs/common/test/unit/services/DidDocumentService.test.ts

```diff
@@ -28,6 +28,14 @@ const encryptedSeedElem =
   '26fd7d484102945b986b724d0ffbe49894a410d2592046b1b6' +
   'f6d18b619e97a2033f0ec7a6630fdcd3827a0dd70b3c439ab4' +
   '76b3fc264b639c84935d6e5d6fcabb3d027d411ae5d74d570fd16d604b038a9250ce4ac271fd6a86d8401ac52c52'
+const encryptedSeedElemAnchored =
+  '58a387ea33ffa071707a4e3fdf4e9042f11e8512fcc51193b066' +
+  '73c3f7b6c7edf7acdb2ee70eda9c69f39640b472cb38c78bb661dd' +
+  '7bdf1cefbf5eefb30625a3eb6b64776238cf8585edea5410f956566cb0' +
+  '16f6a567ee9561801c0f12b445aa5c9255aa6c68842bb82ae1c7a6c823c0a' +
+  '4f6c4f50a420e18c4513b226e798d9bfbd4ec746f45a0d3a7b43e09ac1348332' +
+  'a3a54748a0f8b5ea917c273f7303be545dde3b95ac6dc5ded4f548fca00bbcc5e8' +
+  '00a61b0f36885ce50e986370045c190f6054c0bf1992740ebaf2874eb7f6f'
```

Comment:
Is there any logic to it being splitted into lines in this way and not another?

Reply:
Good question, related to prettier I think. Is it make important  from your side?

Reply:
Prettier would have probably suggested to split the large line into equal-length chunks (probably 120 characters long), but here all chunks are of wildly different lengths (58, 60, 64, 67, 70, 72 and 65). Is it meaningful?

Reply:
Anyway I'll change the elemAnchoredSeed, so copy existing approach